arangodb-net-standard/Serialization/IApiClientSerialization.cs

```diff
@@ -7,6 +7,11 @@ namespace ArangoDBNetStandard.Serialization
     /// </summary>
     public interface IApiClientSerialization
     {
+        /// <summary>
+        /// The default serialization options.
+        /// </summary>
+        IApiClientSerializationOptions DefaultOptions { get; }
```

Comment:
Why expose this in the interface if it's not editable? It might make sense to have `DefaultOptions` declared in the abstract class but here it doesn't seem to provide any value. To me, it is implementation detail rather than a specification in the contract.